### DISCUSSION TITLE
Is it possible to obtain the 99% CI of effect sizes?

### DISCUSSION THREAD

**Discussion by shiqiluo520** [FrederickHuangLin/ANCOMBC#107] [Q&A]
Hi @FrederickHuangLin，

I am trying to calculate the 99% CI of effect sizes, and found the script for Fig.6 # Test result,

```
alpha.adj=0.05/nrow(res.ANCOM_BC)
critic.val=qnorm(1-alpha.adj/2)
res.ANCOM_BC=res.ANCOM_BC%>%transmute(phylum,
                                       `log fold change (MA - US)` = -`mean.difference (US - MA)`,
                                       se = `se (US - MA)`,
                                       ci.lo.adj=`log fold change (MA - US)`-critic.val*se, 
                                       ci.up.adj=`log fold change (MA - US)`+critic.val*se, 
```

But I am not sure whether it is correct to change the "0.05"to "0.01". If not, would you please tell me how can I do it?

Thank you!

Bests,
Shiqi

**Comment by FrederickHuangLin** [comment]
Hi @shiqiluo520,

Yes, you can do it, but please keep in mind that these are **Bonferroni Corrected** Confidence Intervals. There is no straightforward way, to my best knowledge, to construct confidence intervals if you used other multiple correction methods.

Best,
Huang

**Reply by shiqiluo520** [reply]
Thank you！

Bests,
Shiqi
